parsers/LK.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+
+"""Hourly data parser for Sri Lanka
+Fetches data for the previous day in 15-minute increment
+Data is from the backend for the load curve graph on https://cebcare.ceb.lk/gensum/details
+"""
+
+import logging
+# The arrow library is used to handle datetimes
+import arrow
+# The request library is used to fetch content through HTTP
+import requests
+import json
+from parsers.lib.exceptions import ParserException
+
+TIMEZONE_NAME = 'Asia/Colombo'
+GENERATION_BREAKDOWN_URL = "https://cebcare.ceb.lk/GenSum/GetLoadCurveData"
+SOURCE_NAME = "ceb.lk"
+
+def fetch_production(zone_key='LK', session=None, target_datetime=None, logger: logging.Logger = logging.getLogger(__name__)):
+  """Requests the last known production mix (in MW) of a given region
+  Arguments:
```

Comment:
Let's get rid of this comment from the example :)

Reply:
Done